metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
They are not required for ssl to work, it just adds extra security (and is recommended). But they are allowed to be public (since they are transmitted during the exchange). So perhaps better is to add one as a file which is then copied during charm installation to a fixed directory?